client-http/src/main/java/com/cloudera/livy/client/http/LivyConnection.java

```diff
@@ -123,10 +128,6 @@ public Principal getUserPrincipal() {
       };
     }
 
-    // This is needed to get Kerberos credentials from the environment, instead of
-    // requiring the application to manually obtain the credentials.
-    System.setProperty("javax.security.auth.useSubjectCredsOnly", "false");
-
```

Comment:
Now `javax.security.auth.useSubjectCredsOnly` is only set when spnego is enabled. While previously we set it in any case. Would this affect the non-secure mode ?